pkg/gather/ssh/agent.go

```diff
@@ -37,18 +39,19 @@ func newAgent(keyPaths []string) (agent.Agent, error) {
 
 	ag := agent.NewKeyring()
 	var errs []error
-	for idx := range keys {
-		if err := ag.Add(agent.AddedKey{PrivateKey: keys[idx]}); err != nil {
-			errs = append(errs, errors.Wrap(err, "failed to add key to agent"))
+	for name, key := range keys {
+		if err := ag.Add(agent.AddedKey{PrivateKey: key}); err != nil {
+			errs = append(errs, errors.Wrapf(err, "failed to add %s to agent", name))
 		}
+		logrus.Debugf("Added %s to agent", name)
```

Comment:
`Added {} to agent`, since `agent` is not user'd global agent, rather a local agent we have created, `Added {} to local agent` might be more appropriate ?

Reply:
> `Added {} to local agent` might be more appropriate ?

Went with `Added %s to installer's internal agent` in 90064da41 -> 33e334d37, to avoid folks confusing "local" as "the local machine, which might have a few agents", when we mean "not any agents outside the installer".